app.json

```diff
@@ -16,8 +16,10 @@
     "ORIENTATION_LOGO": "/assets/default_logo.svg",
     "ORIENTATION_FAVICON": "/assets/default_favicon.png",
     "ORIENTATION_DOMAIN": "REDACTED",
-    "GOOGLE_SECRET": "REDACTED",
-    "GOOGLE_KEY": "REDACTED",
+    "OAUTH_PROVIDER_NAME": "REDACTED",
+    "OAUTH_PROVIDER_SECRET": "REDACTED",
+    "OAUTH_PROVIDER_KEY": "REDACTED",
+    "OAUTH_PROVIDER_URL": "REDACTED",
```

Comment:
Sadly we can't rename env vars like this, it would break every Orientation installation out there for no obvious reason. We need to support the existing envs as a fallback.